etna/records/blocks.py

```diff
@@ -98,5 +98,8 @@ def value_from_form(self, value):
     def get_form_state(self, value):
         return self.widget.get_value_data(value)
 
+    def extract_references(self, value):
+        return []
```

Comment:
While it's fresh in your head, it'd be great to add a docstring here to explain why this override is needed. Perhaps starting with... ```Overrides ChooserBlock.extract_references() to prevent Wagtail's reference index....```